### DISCUSSION TITLE
Welcome to MinkowskiEngine Discussions!

### DISCUSSION THREAD

**Discussion by chrischoy** [NVIDIA/MinkowskiEngine#349] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by janismdhanbad** [comment]
Hi, this is Janpreet, I am currently a Masters student at The University of Michigan with a Specialization in Computer Vision.  I am working on the reconstruction of point clouds using autoencoders. I was wondering if we could use the coordinates in the sparse tensors to backpropagate on weights? Something like `sout.C` and then using this to compute loss. Thanks!

**Comment by Vikram12301** [comment]
I was trying to locate the file from MinkowskiEngineBackend._C in the repository, but I am unable to locate it. Can someone say where that directory is, as there are many other files dependent on it
